src/pawns.cpp

```diff
@@ -61,6 +61,14 @@ namespace {
   // Unsupported pawn penalty
   const Score UnsupportedPawnPenalty = S(20, 10);
 
+  // Center Bind Bonus: Two pawns controlling the same central square
+  const Bitboard CenterBindMask[] = {
```

Comment:
Please use 

const Bitboard CenterBindMask[COLOR_NB]

to document is indexed by color